gpgwrapper/gpgwrapper.go

```diff
@@ -74,6 +76,23 @@ func (g *GnuPG) Version() (string, error) {
 	return version, nil
 }
 
+// Returns the GnuPG version string, e.g. "1.2.3"git
+func (g *GnuPG) HomeDir() (gpgHomeDir string) {
```

Comment:
Ah, I shoulda been clearer, if you run `gpg --version` it actually prints out its home directory. I think that's the safe way of getting the home directory because can we really second-guess what GPG's gonna do?
```
gpg (GnuPG) 2.2.4
libgcrypt 1.8.1
Copyright (C) 2017 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Home: /home/paul/.gnupg
Supported algorithms:
Pubkey: RSA, ELG, DSA, ECDH, ECDSA, EDDSA
Cypher: IDEA, 3DES, CAST5, BLOWFISH, AES, AES192, AES256, TWOFISH,
        CAMELLIA128, CAMELLIA192, CAMELLIA256
Hash: SHA1, RIPEMD160, SHA256, SHA384, SHA512, SHA224
Compression: Uncompressed, ZIP, ZLIB, BZIP2
```

Reply:
Ah, ok, cool - I'll use that. Dammit. I scoured the *man* page and GnuPG docs and it didn't make that clear at all. Grrrr! 😈

Reply:
Ha sorry about that!